resources/ext/config/conf.d/puppetserver.conf

```diff
@@ -74,3 +74,9 @@ profiler: {
     # enable or disable profiling for the Ruby code; defaults to 'true'.
     #enabled: true
 }
+
+# settings related to the certificate authority
+certificate-authority: {
```

Comment:
Should we put this in its own file?

Reply:
hm. Do you mean like a ``certificate_authority.conf`` or am I misunderstanding?

Reply:
I think the docs tend to refer to a `ca.conf`.

Reply:
ah, that makes more sense. Should I move it then?

Reply:
Jayant's also adding things to the `certificate-authority` section and I *think* it's going to be a thing that warrants its own, albeit small, file by the time we ship Puppet 6. I'm a little torn on whether its better to do that now or if I'm setting us up for YAGNI.

Genuinely unsure on that.